index.js

```diff
@@ -30,11 +25,14 @@ AnimatedGifDetector.prototype.isAnimated = function(buffer) {
   var result = false
     , count = 0
   ;
-  for (var i = 0; i < buffer.length; i++) {
+  var packed = buffer.slice(10, 11)[0];
+  var colorTableSize = 3*2^(packed+1);
+  var leadBytes = 13 + colorTableSize;
+  for (var i = leadBytes + 1; i < buffer.length; i++) {
```

Comment:
Can remove this if you like, but it just calculates how big the global color table is, and doesn't bother searching through that header space.